### ISSUE TITLE
Show delete draft project option on project details page

### ISSUE THREAD

**Issue by divine-comedian** [Giveth/giveth-dapps-v2#4838]
Now the "delete" option is only shown in my project list, shouldn't this option also be shown in project page ? like as continu creation button a button there should be for delete?
@mosaeedi @divine-comedian 

![image](https://github.com/user-attachments/assets/534bdb4b-b899-490e-8d27-0b6364790989)

_Originally posted by @maryjaf in https://github.com/Giveth/giveth-dapps-v2/issues/1271#issuecomment-2401971253_

**Comment by divine-comedian** [comment]
Yes and we should think of what happens when the user deletes the project while they are on this view - where are they redirected to once they delete the project they are currently viewing? 

My suggestion is they are redirected back to My Projects view

**Comment by kkatusic** [comment]
@divine-comedian will I implement this?

**Comment by divine-comedian** [comment]
let's hold off on this work for now.
